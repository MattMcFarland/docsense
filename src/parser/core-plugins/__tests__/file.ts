import filePlugin from '../file';
import loadPlugin from './utils/loadPlugin';

describe('core-plugin: file', () => {
  test('matches snapshot', async () => {
    const runTest = loadPlugin({}, '__TEST__');
    const state = await runTest(filePlugin, 'foo()');
    expect(state).toMatchSnapshot();
  });
});
