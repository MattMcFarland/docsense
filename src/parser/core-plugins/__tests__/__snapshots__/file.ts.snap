// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`core-plugin: file matches snapshot 1`] = `
Object {
  "file_collection": Array [
    Object {
      "base": "__TEST__",
      "dir": "",
      "ext": "",
      "id": "L19fVEVTVF9f",
      "isIndex": false,
      "kind": "Other",
      "name": "__TEST__",
      "path": "__TEST__",
    },
  ],
  "sourceCode": "foo()",
}
`;
